ui/job-view/details/PinBoard.jsx

```diff
@@ -583,36 +583,40 @@ class PinBoard extends React.Component {
               </button>
               <ul className="dropdown-menu save-btn-dropdown-menu">
                 <li
-                  className={!isLoggedIn ? 'disabled' : ''}
                   title={
                     !isLoggedIn ? 'Not logged in' : 'Repeat the pinned jobs'
                   }
                 >
-                  <a
-                    className="dropdown-item"
+                  <button
+                    className={`${!isLoggedIn ? 'disabled' : ''} dropdown-item`}
                     onClick={() => !isLoggedIn || this.retriggerAllPinnedJobs()}
+                    type="button"
```

Comment:
The reactstrap Button component already includes the `type="button"` attribute (for future reference).

Reply:
Hmm, I think I like your idea of using Reactstrap Buttons instead.  I'll switch them.  I gotta keep things like this in mind more...  :)